signin/signin.css

```
@@ -0,0 +1,150 @@
+@import "palette.css";
+
+*{
+  box-sizing : border-box;
+}
+html{
+  height: 100%;
+}
+body{
+  display: flex;
+  width: 100%;
+  padding: 238px 0px 253px 0px;
+  justify-content: center;
+  align-items: center;
+  background: var(--Linkbrary-bg, #F0F6FF);
+}
+.header{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 16px;
+  margin-bottom: 30px;
+}
+.header .container{
+  gap: 8px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.header p{
+  margin: 0;
+  color: var(--black, #000);
+  font-family: Pretendard;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px; /* 150% */
+}
+.header a{
+  color: var(--Linkbrary-primary-color, #6D6AFE);
+  font-family: Pretendard;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: normal;
+}
+.form-container{
+  gap:24px;
+}
+.login-form label{
+  color: var(--black, #000);
+  font-family: Pretendard;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: normal;
+}
+.login-form input{
+  display: flex;
+  width: 400px;
+  padding: 18px 15px;
+  justify-content: center;
+  align-items: center;
+
+  border-radius: 8px;
+  border: 1px solid var(--Linkbrary-primary-color, #6D6AFE);
+  background: var(--Linkbrary-white, #FFF);
+}
+.login-form .wrapper{
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 12px;
+  margin-bottom: 24px;
+}
+.login-form .login-button{
+  margin-top: 30px;
+  display: flex;
+  width: 400px;
+  padding: 16px 20px;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+
+  border-radius: 8px;
+  background: var(--gra-purpleblue-to-skyblue, linear-gradient(91deg, #6D6AFE 0.12%, #6AE3FE 101.84%));
+  border: 0;
+}
+
+.social-login{
+  margin-top: 32px;
+  display: flex;
+  width: 400px;
+  padding: 12px 24px;
+  justify-content: space-between;
+  align-items: center;
+
+  border-radius: 8px;
+  border: 1px solid var(--Linkbrary-gray20, #CCD5E3);
+  background: var(--Linkbrary-gray10, #E7EFFB);
+}
+.social-login .icon{
+  display: flex;
+  align-items: flex-start;
+  gap: 16px;
+}
+.social-login p{
+  color: var(--Linkbrary-gray100, #373740);
+  font-family: Pretendard;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: normal;
+}
+
+
+
+/*오류메세지 처리 태그 기본값 opacity:0*/
+.form-container .wrapper{
+  position: relative;
+}
+
+.form-container #pw-error-msg{
+  position: absolute;
+  bottom:-40px;
+  left:5px;
+  opacity: 0;
+}
+.form-container #email-error-msg{
+  position: absolute;
+  bottom:-40px;
+  left:5px;
+  opacity: 0;
+}
```

두 요소의 스타일이 동일하기 때문에 id로 스타일을 제어하기 보다는 하나의 클래스로 제어하는게 좋아보여요.
```suggestion
.form-container .error-msg{
  position: absolute;
  bottom:-40px;
  left:5px;
  opacity: 0;
}
```